README.md

```diff
@@ -55,9 +55,44 @@ You can then use our tracing dashboards to visualize the requests as traces, whi
 
 ## Send Metrics Data
 
-If your application uses an OpenTelemetry SDK, you can configure the application to send metrics data to Tanzu Observability using the OpenTelemetry Collector and the Wavefront proxy. Metrics data includes time series, counters, and histograms. When the data is in Tanzu Observability, you can use charts and dashboards to visualize the data and create alerts.
+If your application uses an OpenTelemetry SDK, you can configure the application to send trace data to Tanzu Observability using any of the following options listed below. Metrics data includes time series, counters, and histograms. When the data is in Tanzu Observability, you can use charts and dashboards to visualize the data and create alerts.
 
-Here's how it works:
+### Directly Send Data Using the Wavefront Proxy - (Recommended)
+<img src="images/opentelemetry_proxy_tracing.png" alt="A data flow diagram that shows how the data flows from your application to the proxy, and then to Tanzu Observability" style="width:750px;"/>
+
+Follow these steps:
+
+1. [Install the Wavefront Proxy](https://docs.wavefront.com/proxies_installing.html) version 12 or higher.
+1. Proxy configurations
+    * Open the port on the Wavefront Proxy to send OpenTelemetry spans to Tanzu Observability. 
+      * Port 4317 (recommended) with `otlpGrpcListenerPorts` 
+      * Or port 4318 (recommended) with `otlpHttpListenerPorts`
+      * Ensure that port 2878 is open to send metrics to Tanzu Observability. For example, on Linux, Mac, and Windows, open the <a href="https://docs.wavefront.com/proxies_configuring.html#proxy-file-paths"><code>wavefront.conf</code></a> file and confirm that <code>pushListenerPorts</code> is set to 2878, and that this configuration is uncommented.
```

Comment:
@keep94: I added this line. We need port 2878 for metrics, right? :)

Reply:
correct.